target/linux/ramips/image/mt76x8.mk

```diff
@@ -618,3 +618,15 @@ define Device/zyxel_keenetic-extra-ii
 	check-size | zyimage -d 6162 -v "ZyXEL Keenetic Extra II"
 endef
 TARGET_DEVICES += zyxel_keenetic-extra-ii
+
+define Device/zyxel_keenetic-kn-1210
+  DTS := kn-1210
+  IMAGE_SIZE := 16064k
+  BLOCKSIZE := 64k
+  DEVICE_TITLE := Keenetic 4G (KN-1210)
```

Comment:
Use DEVICE_VENDOR/DEVICE_MODEL instead.

Reply:
Is the "Keenetic 4G" name unique? I'd rather this one for the image name, as the other one is only given in brackets when looking for this device...

Reply:
fix